clients/src/test/java/org/apache/kafka/common/serialization/SerializationTest.java

```diff
@@ -147,6 +147,27 @@ public void floatSerdeShouldPreserveNaNValues() {
         }
     }
 
+    @Test
+    public void testSerializeVoid() {
+        try (Serde<Void> serde = Serdes.Void()) {
```

Comment:
Why do we need a `try` for this case? The `close()` is a no-op anyway. (Similar below)